playground/fibonacci_using_for.js

```diff
@@ -0,0 +1,26 @@
+function fibo (idx) {
+    let sumNum = 0;
+    
+        if (idx == 1 || idx == 2) {
+            return 1
+        } else if( idx >= 3) {
+            let curNum=1 ;
+            let bfNum=1 ;
+            for ( let i=1; i<=idx-2; i++){
+    
+                sumNum = bfNum + curNum;
+                bfNum = curNum;
+                curNum = sumNum;
+    
+            }
+            return sumNum;
+        }
+        
+    }
```

Comment:
```suggestion
}
```

전체적으로 괄호 위치를 맞출 필요가 있을 것 같습니다.